python/src/equistore/block.py

```diff
@@ -224,6 +270,34 @@ def add_gradient(
         """
         Add a set of gradients with respect to ``parameters`` in this block.
```

Comment:
Added information how to add components to a gradient. Also tried to make the first sentence a bit clearer. Do you think it is clear?
```suggestion
        Add a gradient containing gradient values with respect to ``parameter`` to this block.
        The gradient must contain the values components, but can contain additional components.
        These additional components must be prepended to the list of values components, 
        with the values components following.
```
```suggestion
        Get the gradient of the block ``values`` with respect to the
        given ``parameter``. 
      
```

Reply:
we do this in a different PR